### ISSUE TITLE
type: "Python"

### ISSUE THREAD

**Issue by feiyun1265** [xxradon/PytorchToCaffe#60]
pytorch to caffe appears layer：type: "Python" 
But caffe read this layer wrong， The error message is shown below：
Unknown layer type: Python (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Clip, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, Swish, TanH, Threshold, Tile, WindowData)

That's all error , thanks for your help!

**Comment by xxradon** [comment]
There is no python layer will be export,can you show your model code so that I can check out what is wrong?
